addons/point_of_sale/static/src/app/screens/payment_screen/payment_lines/payment_lines.xml

```diff
@@ -56,7 +56,7 @@
                                     <div class="electronic_status">
                                         Waiting for card
                                     </div>
-                                    <div class="button send_payment_cancel" title="Cancel Payment Request" t-on-click="() => this.props.sendPaymentCancel(line)">
+                                    <div t-if="!this.props.isRefundOrder" class="button send_payment_cancel" title="Cancel Payment Request" t-on-click="() => this.props.sendPaymentCancel(line)">
```

Comment:
can't we cancel refund payment request?

Reply:
No, there is no request to cancel a refund/void request because refund/void approval comes instantly no need to wait.